src/Log/LogService.php

```diff
@@ -417,10 +425,21 @@ public function getDefaultFormat()
             $this->defaultFormat = $format;
         }
 
-        $runId = $this->getRunId();
+        $format = $this->addRunIdToFormat($this->defaultFormat);
```

Comment:
I think we should not do this here anymore. The `getDefaultFormat` function should simply return the template as it was set using `setDefaultFormat` or as it was configured. Makes more overall sense.

Reply:
Then some tests will have to be changed accordingly.